crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -278,23 +278,69 @@ pub struct StripePaymentIntentResponse {
     pub mandate_id: Option<String>,
     pub metadata: Option<Value>,
     pub charges: Charges,
+    pub connector: Option<String>,
+    pub description: Option<String>,
+    pub mandate_data: Option<payments::MandateData>,
+    pub setup_future_usage: Option<api_models::enums::FutureUsage>,
+    pub off_session: Option<bool>,
+    pub return_url: Option<String>,
+    pub authentication_type: Option<api_models::enums::AuthenticationType>,
+    pub next_action: Option<payments::NextAction>,
+    pub cancellation_reason: Option<String>,
+    pub payment_method: Option<api_models::enums::PaymentMethodType>,
+    pub payment_method_data: Option<payments::PaymentMethodDataResponse>,
+    pub shipping: Option<payments::Address>,
+    pub billing: Option<payments::Address>,
+    pub capture_on: Option<time::PrimitiveDateTime>,
```

Comment:
What about the date time format to be used here? ISO 8601 or Stripe compatible?

Reply:
This is not a stripe compatible change. If the core sends a field in response and stripe doesn't send it. I still send it over here. The merchant can use the value or leave it.

The fields which are stripe compatible are converted to the appropriate type and sent.

Reply:
Then I guess we should keep it compatible with our response format at least, so please serialize it using ISO 8601 format.